kyo-data/shared/src/test/scala/kyo/DurationSpec.scala

```diff
@@ -157,7 +157,7 @@ object DurationSpec extends ZIOSpecDefault:
                 )
             }
         )
-    )
+    ) @@ TestAspect.exceptNative
```

Comment:
The tests actually pass but I think there's some incompatibility in the test runner that makes it report a failure.